### PR TITLE
fix transparent edge during side menu transition

### LINKED ISSUE #57: Side menu shouldn't reveal transparent area on side during animation
Should extend side menu width by ~10px but make it stop at ~-10px.

### PR DESCRIPTION
**Pull request by taylordolan**
fixes #57

### CODE REVIEW

File: app/components/SideMenu.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ const styles = StyleSheet.create({
   },
   menu: {
     flex: 1,
-    width: DeviceScreen.width - MENU_OFFSET,
+    width: DeviceScreen.width - MENU_OFFSET + 20,
```
Comment (philipcdavis) [inline app/components/SideMenu.js]: I think we could add this to the MENU_OFFSET constant. 

@BrettBukowski is that ok?

Comment (learningd3) [inline app/components/SideMenu.js]: I think we can add this to the MENU_OFFSET constant.

@BrettBukowski is that ok?

**Comment by learningd3** [comment]
Outside of my comment, this looks good to me!

**Comment by BrettBukowski** [comment]
lgtm

**Pull request merged**